android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
+eclipse.preferences.version=1
```

Comment:
Why .settings/org.eclipse.buildship.core.prefs is needed?

Reply:
I am sorry. Forgot to add it to .gitignore. It is configuration file of my IDE. Just ignore it